### PR TITLE
Max's Crate Experience

### PR DESCRIPTION
**Pull request by celaus**

### CODE REVIEW

File: pages/a/max-internship-at-crate.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+*To end his internship at Crate, we asked Max to contribute to our blog and tell us about the experience of working at Crate. At 18 years old he attends a technical school in Austria and while internships are sometimes mandatory, the company is free to choose. Luckily - he chose us.*
+
+
+"It would be fun they said, just an other internship they said", but they were so wrong (at least in respect to "just an internship") but let's start at the beginning.
+
+![Max cooking for the Dornbirn office.](/static/images/0816/max-cooking.jpg)
+
+
+After my successful application as an intern I started my six weeks at Crate's Dornbirn office on the 11th of July.
+I was not sure what to expect but the welcoming email raised my expectations and piqued my interest.
```
Comment (therealpadams) [inline pages/a/max-internship-at-crate.html]: This is a "teaser". What was in this welcoming email that made is so fascinating? We should create a more obvious link to the "awesome to the Max" image at the end.

File: pages/a/max-internship-at-crate.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+After I got familiar with setting up and operating a Crate cluster, I started to play around with the [sample apps](https://github.com/crate/crate-sample-apps) Crate.IO provides. The first part of this task was to program my own backend app and to run it locally on my notebook. In order to make this possible I had to learn some Python and read through a lot of web development tutorials which took a few days.
+The implementation of the front end was a little bit easier as, in my opinion, manipulating HTML code is much more intuitive than programming in Python.
+
+Anyhow the opportunity which made this internship really unique was offered to me in the forth week. In the course of the birthday [hackathon for Kubernetes and Crate](https://crate.io/a/happy-birthday-kubernetes) I had the opportunity to work in Berlin for seven days. There not only did I get to know a beautiful city but also worked with a fantastic team of developers who showed me the city in many different ways.
```
Comment (therealpadams) [inline pages/a/max-internship-at-crate.html]: "Anyhow, the opportunity..."

File: pages/a/max-internship-at-crate.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+To be honest at first I was not sure if I was qualified enough to work at Crate and in the beginning it was pretty hard to work trough so many new things and impressions - but with the help of the team I learned so much in these six weeks and I always felt that I was part of the company. I think that not many interns can claim this.
```
Comment (therealpadams) [inline pages/a/max-internship-at-crate.html]: "To be honest,..."

Comment (therealpadams) [inline pages/a/max-internship-at-crate.html]: "...and, in the beginning,..."

File: pages/a/max-internship-at-crate.html (hunk 2 above)
Comment (therealpadams) [inline pages/a/max-internship-at-crate.html]: "...fourth week."

**Comment by celaus** [comment]
@therealpadams approves

**Pull request merged**